Source/gui/SNTBinaryMessageWindowController.h

```diff
@@ -21,7 +21,7 @@
 ///
 ///  Controller for a single message window.
 ///
-@interface SNTBinaryMessageWindowController : SNTMessageWindowController
+@interface SNTBinaryMessageWindowController : SNTMessageWindowController <NSWindowDelegate>
```

Comment:
SNTMessageWindowController already declares that it implements NSWindowDelegate.

Reply:
Apologies this was an accident based on VSCode and delirium from a bad stomach flu. 

Should be fixed.